example/index.html

```diff
@@ -7,6 +7,9 @@
 
   <link href='https://fonts.googleapis.com/css?family=Roboto:300,400,500' rel='stylesheet' type='text/css'>
   <link rel="stylesheet" href="/cms.css"/>
+<link href="config.yml" type="text/yaml" rel="cms-config-url">
```

Comment:
We should remove this from the example project, only because it makes an optional feature seem necessary, and a lot of people treat this example as documentation. We could, however add a comment like:

```html
<!--
  Netlify CMS will automatically look for a "config.yml" file in the same directory
  as this "index.html", but you can override this by providing a link tag like this
  one:
  
  <link href="path/to/config.yml" type="text/yaml" rel="cms-config-url">
-->
```